metapi/metapi.js

```diff
@@ -402,5 +456,19 @@ window.metapi = {};
     sock = sockets[key];
 
     sock.addCallback(messageCallback);
+
+    if (closeCallback) {
+      sock.addCloseCallback(closeCallback);
+    }
+  };
+
+  /**
+   * Registers sockets / listeners for socckets that were requested while an dependency was still loading.
+   */
+  metapi.watchPendingSockets = function () {
```

Comment:
The library does, see line 28.
Although it probably doesn't need to be exposed on `metapi`.